docker-build/run.sh

```diff
@@ -17,4 +17,5 @@ docker run \
   --mount="source=$volumename,target=/data" \
   --publish=9090:9090 \
   --cap-add=sys_nice \
+  $GPU_FLAGS \
```

Comment:
`${GPU_FLAGS:+--gpus=${GPU_FLAGS}}` would imho be better here

So you would set `GPU_FLAGS=all`, while not used when left empty